docker-compose.yaml

```diff
@@ -1,9 +1,27 @@
 version: "3"
 services:
+  tracetest:
+    container_name: tracetest
+    image: kubeshop/tracetest
+    build: .
+    volumes:
+      - ${PWD}/server/config.yaml:/app/config.yaml
+    ports:
+      - 8080:8080
+      - 8081:8081
+    depends_on:
+      - postgres
```

Comment:
Only this doesn't ensure the database will be ready to accept connections. So `docker-compose up` might fail. To prevent that, you can add a [helthcheck routine to the database](https://github.com/kubeshop/pokeshop/blob/master/api/docker-compose.yml#L14) and then [update the depends_on](https://github.com/kubeshop/pokeshop/blob/master/api/docker-compose.yml#L78) configuration

Reply:
I didn't know you could set conditions for depends. thanks!